MainModule/Client/Core/Anti.lua

```diff
@@ -438,9 +438,6 @@ return function(Vargs, GetEnv)
 				"returning json";
 				"shattervast";
 				"failed to parse json";
-				"newcclosure", -- // Kicks all non chad exploits which do not support newcclosure like jjsploit
-				"getrawmetatable";
-				"setfflag";
```

Comment:
The removed detection strings here aren't useless but infact are one of the best ways to detect skidded exploits.
Exploits which do not have the said functions make an error which is sent to the log output. This includes JJSploit and many others